src/helpers/script_helpers.sh

```diff
@@ -153,6 +154,13 @@ function run_steps {
 }
 
 
+# Sets the previous branch after the git town command completes
+# This should be overridden when necessary
+function set_previous_branch {
+  true
+}
```

Comment:
How about about this just returns the branch name that needs to be set as the previous branch? Should help remove some duplication. (may mean renaming the function)